crates/driver/src/demo.rs

```diff
@@ -229,10 +229,9 @@ pub fn read_print_loop(verbosity: Verbosity) {
             }
             Ok(script) => {
                 let program = arena::alloc(allocator, Program::Script(script.unbox()));
-
                 handle_script(
                     &verbosity,
-                    &program.unbox(),
+                    &*program,
```

Comment:
Just `&program` should work. That's what we should have been doing here to begin with.